src/test/java/net/sf/jabref/logic/formatter/OrdinalsToSuperscriptFormatterTest.java

```diff
@@ -2,29 +2,15 @@
 
 
 import net.sf.jabref.logic.formatter.bibtexfields.OrdinalsToSuperscriptFormatter;
-import org.junit.After;
 import org.junit.Assert;
-import org.junit.Before;
 import org.junit.Test;
 
+/**
+ * Tests in addition to the general tests from {@link net.sf.jabref.logic.formatter.FormatterTest}
+ */
 public class OrdinalsToSuperscriptFormatterTest {
-    private OrdinalsToSuperscriptFormatter formatter;
 
-    @Before
-    public void setUp() {
```

Comment:
Quoting [Junit's JavaDoc](http://junit.sourceforge.net/javadoc/org/junit/Test.html) (referenced from http://stackoverflow.com/a/16714168/873282).

> To run the method, JUnit first constructs a fresh instance of the class then invokes the annotated method

Thus, the effects should be the same and my proposal the one with less LOC for the same effects.